src/fairchem/core/_cli_hydra.py

```diff
@@ -131,8 +156,8 @@ def main(
                 rdzv_backend="c10d",
                 max_restarts=0,
             )
-            elastic_launch(launch_config, runner_wrapper)(cfg, args)
+            elastic_launch(launch_config, runner_wrapper)(cfg)
```

Comment:
Just out of curiosity is the `runner_wrapper` just to make `elastic_launch` happy?

Reply:
yep, it can only deal with a top level function